x-pack/plugins/ml/common/util/alerts.ts

```diff
@@ -51,3 +52,27 @@ export function getTopNBuckets(job: Job): number {
 
   return Math.ceil(narrowBucketLength / bucketSpan.asSeconds());
 }
+
+/**
+ * Returns tests configuration combined with default values.
+ * @param config
+ */
+export function getResultJobsHealthRuleConfig(config: JobsHealthRuleTestsConfig) {
+  return {
+    datafeed: {
```

Comment:
Is this key used in the alert column? Wondering if the UI text should be more descriptive here, like `datafeed not started`.

![image](https://user-images.githubusercontent.com/7405507/126334571-3017f845-c69f-4aed-ad61-ed18dc9d92bd.png)

Reply:
I believe we should not care much about names in this UI because it's just an alert name for the framework tracking for updating the status. Renamed it just in case in https://github.com/elastic/kibana/pull/106084/commits/817fd62905500bc1c147694e283d22624d294c7f